src/test/resources/json/html.json

```diff
@@ -0,0 +1,71 @@
+[
```

Comment:
BTW, you can rebase code

App is working correctly now (when you removed lib dependency)